src/WebJobs.Script/Binding/WebJobsCoreScriptBindingProvider.cs

```diff
@@ -92,6 +58,7 @@ public override bool TryCreate(ScriptBindingContext context, out ScriptBinding b
             {
                 binding = new ManualScriptBinding(context);
             }
+            // Queue, Table handled by general case. 
```

Comment:
nit: remove - this is more of a CR comment, and doesn't need to be in the source I think